designsafe/static/scripts/projects/components/confirm-message/confirm-message.component.js

```diff
@@ -10,7 +10,7 @@ class ConfirmDeleteCtrl {
     }
 
     $onInit() {
-        this.ent = this.resolve.options.entity;
+        this.message = this.resolve.message;
     }
 
     delete () {
```

Comment:
This function might also need to forward the user back to the project listing.  As it stands now, after I remove myself from a project I'm returned to the project.  And, I still have the ability to add myself back as a PI.